### DISCUSSION TITLE
Handle cuesheets, either embedded in a Flac or as a separate file.

### DISCUSSION THREAD

**Discussion by Particpant** [music-assistant/support#2305] [Feature requests and ideas]
Right now, a library of single file flac albums with either embedded or separate cuesheets are discovered as an album with a single track, with the length of the full first track. It'd be great if these can be handled like LMS does (recognize the track listing and present the ability to play the individual tracks or skip between them.)

**Comment by OzGav** [comment]
Whilst this was first this other duplicate has more votes so cloing this one https://github.com/orgs/music-assistant/discussions/2811
